persistence-api/src/main/java/io/stargate/db/Persistence.java

```diff
@@ -252,5 +233,16 @@ default void setCustomProperties(Map<String, String> customProperties) {}
 
     /** Makes a new {@link RowDecorator} for the given table. */
     RowDecorator makeRowDecorator(TableName table);
+
+    /**
+     * Checks whether this coordinator in schema agreement with the other nodes in the cluster. This
+     * is the same as {@code Persistence#isInSchemaAgreement()}, except that it *may* consider other
+     * connection specific properties to determine whether schema is in agreement.
+     *
+     * @return true if schema is agreement; otherwise false.
+     */
+    default boolean isInSchemaAgreement() {
+      return persistence().isInSchemaAgreement();
```

Comment:
this would actually guarantee the same old behavior by default here.. But any `Connection` impl can provide it's own version.. Nice :+1: